src/models/metadata.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const config = require('../services/config')
+
+function getMetadata () {
+  return {
+    public_key: config.getIn(['keys', 'ed25519', 'public']),
+    urls: {
```

Comment:
The "urls" object should probably be added by the controller since it's an HTTP thing. And for clarity this method could be renamed to something like "getDomainMetadata", which then gets merged with HTTP metadata.